components/sensor/pulse_meter.rst

```diff
@@ -24,85 +32,80 @@ Configuration variables:
 - **name** (**Required**, string): The name of the sensor.
 
 - **internal_filter** (*Optional*, :ref:`config-time`): If a pulse shorter than this
-  time is detected, it’s discarded and no pulse is counted. Defaults to ``13us``. For S0 pulse meters that are used to meter power consumption 50-100 ms is a reasonable value.
+  time is detected, it’s discarded and no pulse is counted. Defaults to ``13us``. 
+  
+  To calculate a meaningful value, you need to know the shortest pulse you expect to see, as not to filter out valid pulses.
 
 - **internal_filter_mode** (*Optional*, string): Determines how the internal filter is applied.
-  One of ``EDGE`` and ``PULSE``. Defaults to ``EDGE``. In ``EDGE`` mode subsequent rising edges are compared and if they fall into an interval lesser than the internal filter value, the last one is discarded. In ``PULSE`` mode the rising edge is discarded if any further interrupts are detected before the internal_filter time has passed. In other words, a high pulse must be at least internal_filter long to be counted. This is useful if you are detecting long pulses that may bounces before and/or after the main pulse.  
+  One of ``EDGE`` and ``PULSE``. Defaults to ``EDGE``. In ``EDGE`` mode subsequent rising edges are compared and if they fall into an interval lesser than the internal filter value, the last one is discarded. In ``PULSE`` mode the rising edge is discarded if any further interrupts are detected before the ``internal_filter`` time has passed. In other words, a high pulse must be at least ``internal_filter`` long to be counted. This is useful if you are detecting long pulses that may bounces before and/or after the main pulse.  
 
-- **timeout** (*Optional*, :ref:`config-time`): If we don't see a pulse for this length of time, we assume 0 pulses/sec. Defaults to ``5 min``.
+- **timeout** (*Optional*, :ref:`config-time`): If we don't see a pulse for this length of time, we assume ``0 pulses/s``. Defaults to ``5 min``.
 - **total** (*Optional*, :ref:`config-id`): An additional sensor that outputs the total number of pulses counted.
 - All other options from :ref:`Sensor <config-sensor>`.
 
 Converting units
 ----------------
 
-The sensor defaults to units of “pulses/min”. You can change this by using :ref:`sensor-filters`.
-For example, if you’re using the pulse meter with a photodiode to
-count the light pulses on a power meter that outputs 1000 pulses per kWh,
-you can use the following to output instantaneous usage in kW:
+The sensor defaults to units of ``pulses/min``. You can change this by using :ref:`sensor-filters`.
```

Comment:
It depends on the meter, the two modern ones I've seen in Germany, which admittedly were 3 phase meters, had 10000/kWh. 

Browsing through image results of smart meters has given results such as 400 and 600 pulses also so it's fairly arbitrary. I went with what I had because I wanted to avoid an accidental miscalculation or the like.